command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 11:45:46AM -0700, Doug Davis wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
> 
> Let's change this to something like:
> ```
> A conformant runtime MAY expose a command line interface/executable
> (called `funC` in the following examples).
> ```
> Not all conformant runtime need to support a cli - they could be
> embedded within a bigger system but still be compliant to the
> runtime spec.

I think the point of this PR is that runtimes *do* need to provide a
command line interface, and that compliance testing will be based on
driving the runtime through that interface (e.g. in ocitools).
Compliant runtimes are free to provide other interfaces as they see
fit, and the command they provide for compliance testing may be a
wrapper/shim around their native API.  More discussion on this point
at [1] and later.

[1]: https://github.com/opencontainers/runtime-spec/pull/513#issuecomment-231609551